src/pages/Profile.tsx

```diff
@@ -145,11 +146,15 @@ const Profile = () => {
           <h1 tw="font-extra-medium">
             {isConnected ? ensName ?? truncateAddress(dydxAddress) : '-'}
           </h1>
-          {isConnected && walletType ? (
+          {isConnected && connectedWallet ? (
```

Comment:
Should we be using `selectedWallet` instead? `connectedWallet` is used as the initial value for `selectedWallet` and may not be populated if the user is using a browser that doesn't support `localStorage` (i.e. incognito mode)

Reply:
seems like it yeah 👀 i was mostly copying the behavior from before, where `walletType` was the one that was persisted and read from localStorage

Reply:
gotcha. yeah, it seems like quite a few places where we are using this. We should just hope that the `selectedWallet` and `connectedWallet` are always in sync

Reply:
should we full send it and just change everything to the `selectedWallet` version 👀

Reply:
I think we should. Are there any user flows where they would become out of sync?

Reply:
talked offline and we r indeed sticking with connectedWallet 🤝